pandas/tests/test_graphics.py

```diff
@@ -183,7 +193,7 @@ def _check_colors(self, collections, linecolors=None, facecolors=None,
         """
 
         from matplotlib.lines import Line2D
-        from matplotlib.collections import Collection
+        from matplotlib.collections import Collection, PolyCollection
```

Comment:
For reference, mpl does not support past the current minor release (ex, we currently do not fix bugs in 1.3.x, and in a short while will not fix bugs in 1.4.x).